test/index.js

```diff
@@ -189,6 +189,9 @@ describe("getEnvironmentVariables", () => {
 		let originalShellEnv = process.env.SHELL;
 		process.env.SHELL = shellEnv;
 
+		let fs = require("fs");
+		fs.existsSync = (filePath) => false;
```

Comment:
I think it'd be reasonable to save the original `existsSync` function and revert it in the `afterEach` like you do with `fs.readFileSync` and `fs.exists`